tests/lean/cyclic_default_fields.lean

```diff
@@ -1,7 +1,6 @@
 class Eq (α : Type) :=
-(eq : α → α → Prop)
+(eq : α → α → Prop := λ a b, ¬ne a b)
 (ne : α → α → Prop := λ a b, ¬eq a b)
```

Comment:
Realized this only while working on the PR, even though they are completely independent :) .